src/madness/world/dqueue.h

```diff
@@ -78,9 +85,18 @@ namespace madness {
         volatile int _back;    ///< Index of element at back of buffer
         DQStats stats;
 
+#ifdef MADNESS_DQ_USE_PREBUF
+	static const size_t NPREBUF=20;
```

Comment:
presumably the optimal is application (or better yet op-dependent) so do we want runtime control of the granularity? Our tasks are fairly fat so I'd imagine we don't want this param as high ... at least make it a CMake-configurable value

Reply:
Making this runtime configurable will slow down the code so I left it is compile time configurable.

Reply:
Due to the need to fetch a line to compare ninprebuf{,hi} to this?

OK, can be addressed later.